### DISCUSSION TITLE
Consider offering alternate views when data isn't available

### DISCUSSION THREAD

**Discussion by benhammondmusic** [SatcherInstitute/health-equity-tracker#3857] [HET Ideas (Features, Requests, New Data, etc)]
We should try guiding the user to the info we DO have, rather than just saying we DON’T have it and letting them blindly click around until they find it. For instance if we don’t have the data visualization at the county level, I think they should easily be able to see that we do have it at the state level and easily click to switch views. Same with the demographic and data toggles; if something (age adjustment ratios, share of unknown map, etc) exists in an alternate filtering state, it’d be really nice to be able to guide them up or over a level so it’s available

**Comment by benhammondmusic** [comment]
Semi-related; @joshzarrabi had an excellent idea we should explore to show a STATE map for COUNTY level reports, and highlight the selected county. This accomplishes a few things:
- the county level MAP isn't super helpful as it doesn't compare any sub-geos
- it allows the user to easily click between counties in a state to see info for that county; keeping the map-as-a-UI concept in place
- often we are missing data at a county level that exists at the state leve. There is a reasonable fear that users are seeing a gray "no data" map and immediately leaving the report, not realizing there is more data available below lower on the report

**Comment by jgonzalezmsm** [comment]
I will start a definition document on this. If you agree, we work on this as one of those "as time allows" projects whenever you can fit it in - but let's agree on size and scope. I'll share the doc once I have it.
